endpoints/schemas/commonSchemas.yaml

```diff
@@ -56,6 +56,55 @@ components:
           type: string
           enum: [Base64]
 
+    subscriptionRequest:
+      description: A client creates a subscription by issuing a request against the subscriptions resource. The subscription namespace is client-defined via the POST operation.
+      required:
+        - resource
+        - notificationUrl
+      properties:
+        resource:
+          description: Resource path to watch
+          type: string
+          format: uri
+        notificationUrl:
+          description: The target web hook URL
+          type: string
+          format: uri
+        clientState:
+          description: Opaque string passed back to the client on all notifications. Callers may choose to use this to provide tagging mechanisms.
+          type: string
+
+    subscriptionResponse:
+      description: If the subscription was successfully created, the service MUST respond with the status code 201 CREATED and a body containing at least the following properties
+      required:
+        - id
+        - expirationDateTime
+      properties:
+        id:
+          description: Unique ID of the new subscription that can be used later to update/delete the subscription.
+          type: string
+        expirationDateTime:
+          description: The target web hook URL
+          type: string
+          format: date-time
```

Comment:
any chance we can specify the exact format of date-time? I'd recommend `ISO 8601`

Reply:
@vadimi It appears that the YAML standard  uses the ISO-8601 standard. https://symfony.com/doc/current/components/yaml/yaml_format.html